src/backend/distributed/commands/multi_copy.c

```diff
@@ -1430,7 +1516,8 @@ AppendCopyBinaryFooters(CopyOutState footerOutputState)
  * opens connections to shard placements.
  */
 static void
-StartCopyToNewShard(ShardConnections *shardConnections, CopyStmt *copyStatement)
+StartCopyToNewShard(ShardConnections *shardConnections, CopyStmt *copyStatement,
+					bool binaryCopyFormat)
```

Comment:
this is a really small nitpick feel free to ignore it but something like `useBinaryCopyFormat` sounds slightly better